plugins/inputs/tengine/tengine.go

```diff
@@ -0,0 +1,335 @@
+		}
+		tenginestatus.req_total, err = strconv.ParseUint(line_split[4], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_2xx, err = strconv.ParseUint(line_split[5], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_3xx, err = strconv.ParseUint(line_split[6], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_4xx, err = strconv.ParseUint(line_split[7], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_5xx, err = strconv.ParseUint(line_split[8], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_other_status, err = strconv.ParseUint(line_split[9], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.rt, err = strconv.ParseUint(line_split[10], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.ups_req, err = strconv.ParseUint(line_split[11], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.ups_rt, err = strconv.ParseUint(line_split[12], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.ups_tries, err = strconv.ParseUint(line_split[13], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_200, err = strconv.ParseUint(line_split[14], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_206, err = strconv.ParseUint(line_split[15], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_302, err = strconv.ParseUint(line_split[16], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_304, err = strconv.ParseUint(line_split[17], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_403, err = strconv.ParseUint(line_split[18], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_404, err = strconv.ParseUint(line_split[19], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_416, err = strconv.ParseUint(line_split[20], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_499, err = strconv.ParseUint(line_split[21], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_500, err = strconv.ParseUint(line_split[22], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_502, err = strconv.ParseUint(line_split[23], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_503, err = strconv.ParseUint(line_split[24], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_504, err = strconv.ParseUint(line_split[25], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_508, err = strconv.ParseUint(line_split[26], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_other_detail_status, err = strconv.ParseUint(line_split[27], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_ups_4xx, err = strconv.ParseUint(line_split[28], 10, 64)
+		if err != nil {
+			return err
+		}
+		tenginestatus.http_ups_5xx, err = strconv.ParseUint(line_split[29], 10, 64)
+		if err != nil {
+			return err
+		}
```

Comment:
Optional: maybe we can use `encoding/csv` from the standard library for this?

Reply:
I fell that is not need to use csv